OpenRA.Mods.Common/Traits/World/ResourceClaimLayer.cs

```diff
@@ -111,6 +102,13 @@ public bool IsClaimedByAnyoneElse(Actor self, CPos cell, out ResourceClaim claim
 				// Same claimer:
 				if (claim.Claimer == self) return false;
```

Comment:
Style nit: While you're at it, please move the return here and in lines 115+ to newlines (like in line 63).